conf/messages.cy

```diff
@@ -99,12 +99,10 @@ valueWeightOfGoods.GreatBritain.error.required = Select yes if the total value o
 valueWeightOfGoods.NorthernIreland.error.required = Select yes if the total value of the goods is less than £873 and they weigh less than 1,000 kilograms
 
 # SearchGoods
-goodsTypeQuantity.Import.title = Pa fath o nwyddau ydych chi’n dod â nhw i mewn i Brydain Fawr?
-goodsTypeQuantity.Import.heading = Pa fath o nwyddau ydych chi’n dod â nhw i mewn i Brydain Fawr?
-goodsTypeQuantity.Export.title = Pa nwyddau ydych yn mynd â nhw allan o Brydain Fawr?
-goodsTypeQuantity.Export.heading = Pa nwyddau ydych yn mynd â nhw allan o Brydain Fawr?
-goodsTypeQuantity.p = Ychwanegwch eich nwyddau yn ôl eu math neu gategori, er enghraifft, electroneg. Gallwch ychwanegu rhagor o nwyddau’n nes ymlaen os oes gennych fwy nag un math.
-goodsTypeQuantity.category = Math o nwyddau
+goodsTypeQuantity.title = Enter the first type of goods
```

Comment:
I think we need another message for 'Enter the next type of goods'

This 'first' should only be displayed when idx == 1